Dockerfile

```diff
@@ -9,41 +9,47 @@ ENV LANG="en_US.UTF-8" \
     SINUS_DIR="/sinusbot" \
     YTDL_BIN="/usr/local/bin/youtube-dl" \
     YTDL_VERSION="latest" \
-    TS3_VERSION="3.0.18.2" \
-    TS3_DL_ADDRESS="http://teamspeak.gameserver.gamed.de/ts3/releases/" \
-    SINUSBOT_DL_URL="https://www.sinusbot.com/dl/sinusbot-beta.tar.bz2"
+    TS3_VERSION="3.1.8" \
+    TS3_DL_ADDRESS="http://dl.4players.de/ts/releases/" \
+    SINUSBOT_DL_URL="https://www.sinusbot.com/dl/sinusbot.current.tar.bz2"
```

Comment:
Is `current` going to be the latest version from now on or will the `beta` version be used again and then be the latest again?

Reply:
Im not sure about that. Currently beta is set to 0.10 versions because they only work with the older TS3 versions so at the moment there are using current to hand out the new versions. 
They also edited their install and update entry in their wiki to current.
https://wiki.sinusbot.com/en:guides:installation:linux#update
But may they change it back to beta for full releases and current to the latest build.